src/internal.c

```diff
@@ -13585,17 +13584,15 @@ int SendCertificate(WOLFSSL* ssl)
         length -= (ssl->fragOffset + headerSz);
 
     maxFragment = MAX_RECORD_SIZE;
+
     if (ssl->options.dtls) {
     #ifdef WOLFSSL_DTLS
         maxFragment = MAX_MTU - DTLS_RECORD_HEADER_SZ
```

Comment:
I noticed this section isn't in the `wolfSSL_GetMaxRecordSize` and I think you are right to leave it out. I would like to see a comment here on this calculation. Example ` - 100`. What is the hard coded 100 for? Would you mind reaching out to John S and asking for some details here. Maybe we can add a comment about this in the code? Thanks

Reply:
Yes, this DTLS section deviated from all the others. 

@ejohnstown,
It look like the 100 bytes is to allow for some more header overhead in the packet. Is that correct? Could this use an existing define?

Reply:
I am adding the comment:
```
        /* The 100 bytes is used to account for the UDP and IP headers.
           It can also include the record padding and MAC if the 
           SendCertificate is called for a secure renegotiation. */ 
        maxFragment = MAX_MTU - DTLS_RECORD_HEADER_SZ
                      - DTLS_HANDSHAKE_HEADER_SZ - 100;
```